### PR TITLE
fix: typos and spelling mistakes.

### PR DESCRIPTION
**Pull request by AyshChaudhary**
## Related Issue 

- Info about the related issue 

Closes: #[issue number that will be closed through this PR]

### Describe the changes you've made

Give a clear description what modifications you have made

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Describe how have you verified the changes made

## Checklist:

- [ ] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly whereever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -2,25 +2,25 @@
 
 1. You need to be **already registered** for **GSSoC ’21** as a participant to contribute.
 2. Registrations are **closed** now. We will try to see if we can bring in new people, but it depends on GSSoC Core Team.
-3. Participants / contributors have to **comment** on issues they would like to work on, and mentors or the PA will assign you.
+3. Participants/contributors have to **comment** on issues they would like to work on, and mentors or the PA will assign you.
```
Comment (praveenscience) [inline Contributing.md:5]: Nopes. Need space.

File: Contributing.md [hunk 2]
```diff
@@ -2,25 +2,25 @@
...
 4. Issues will be assigned on a **first-come, first-serve basis.**
-5. Participants / contributors can also **open their issues**, but it needs to be verified and labelled by a mentor. Please discuss with the team once on [**Discord**](https://discord.com/channels/811724426965811210/815033852745220107) before opening your issues. We respect all your contributions, whether it is an issue or a Pull Request.
-6. Each participant / contributor will be **assigned 2 issues (max)** at a time to work.
-7. Participants are expected to follow **project guidelines** and [**coding style**](https://blog.praveen.science/git-standards-followed-in-our-way-of-spotify-agile-methodolgy/). **Structured code** is one of our top priority.
+5. Participants/contributors can also **open their issues**, but it needs to be verified and labeled by a mentor. Please discuss with the team once on [**Discord**](https://discord.com/channels/811724426965811210/815033852745220107) before opening your issues. We respect all your contributions, whether it is an issue or a Pull Request.
```
Comment (praveenscience) [inline Contributing.md:7]: UK / India English - labelled is right. Nopes.

File: Contributing.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ To start contributing, follow steps mentioned below:
 **3.** You can confirm that this repository is a fork of the original repository by finding the "forked from [praveenscience/Internship-LMS-FrontEnd](https://github.com/praveenscience/Internship-LMS-FrontEnd)" text under the repository name.
 ![Step 3](./docs/Step3.png)
 
-**4.** Click on Clone or download button on the forked repository. Make sure you are cloning your version (GitHub.com/\<your username>/Internship-LMS-FrontEnd) and not the CatsInTech organisation's.
+**4.** Click on Clone or download button on the forked repository. Make sure you are cloning your version (GitHub.com/\<your username>/Internship-LMS-FrontEnd) and not the CatsInTech organization's.
```
Comment (praveenscience) [inline Contributing.md:51]: Use UK / Indian English.

File: Contributing.md [hunk 4]
```diff
@@ -126,11 +126,11 @@ git push -u origin <your_branch_name>
...
-**16.** Add appropriate title and description to your pull request explaining your changes and efforts done.  
+**16.** Add appropriate title and description to your pull request explaining your changes and efforts done.  
 ![Step 16](./docs/Step16.png)
 
 **17.** Click on `Create Pull Request`.
 ![Step 17](./docs/Step17.png)
 
 As always, if you have any questions regarding Git, please do reach out on [Discord](https://discord.com/channels/811724426965811210/815033852745220107).
-Discord is only for GSSoC participants.
+Discord is only for GSoC participants.
```
Comment (praveenscience) [inline Contributing.md:136]: GSoC ≠ GSSoC

**Review by praveenscience — CHANGES_REQUESTED** [review]

**Comment by AyshChaudhary** [comment]
Thanks for pointing out my mistakes .

**Pull request closed without merging**